build.gradle

```diff
@@ -148,22 +164,24 @@ dependencies {
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
 
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: versions.springHystrix
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix-dashboard', version: versions.springHystrix
 
   compile group: 'org.flywaydb', name: 'flyway-core', version: '5.0.7'
   compile group: 'org.postgresql', name: 'postgresql', version: '42.2.2'
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: versions.springBoot
-  compile group: 'org.hibernate', name: 'hibernate-hikaricp', version: '5.2.17.Final'
-  compile group: 'com.vladmihalcea', name: 'hibernate-types-5', version: '2.2.0'
+  compile group: 'org.hibernate', name: 'hibernate-hikaricp', version: '5.3.0.Final'
+  compile group: 'com.vladmihalcea', name: 'hibernate-types-5', version: '2.2.1'
 
   compile group: 'org.ff4j', name: 'ff4j-core', version: versions.ff4j
 
   testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot
 
   integrationTestCompile sourceSets.main.runtimeClasspath
   integrationTestCompile sourceSets.test.runtimeClasspath
+  integrationTestCompile group: 'org.testcontainers', name: 'postgresql', version: '1.7.2'
```

Comment:
how did the CI pass before this change? are integration tests run?

Reply:
as mentioned in description:
> explicitly spec out sourceSets as for some reason magical setup did not recognise correct test output

left source sets as it was before in multiproject setup. but apparently it pointed to test folder and not integration. even though task set looked ok.

yes, integration suite is included in travis script